mmv1/products/compute/RegionBackendService.yaml

```diff
@@ -658,6 +658,7 @@ properties:
         description: |
           Time for which instance will be drained (not accept new
           connections, but still work to finish started).
+        deprecation_message: "connection_draining_timeout_sec default value will be changed to 300 to match default GCP value"
```

Comment:
Terraform deprecations messages are highly disruptive for users and should be reserved for fields that will actually be removed. I quickly conferred with the team, and we don't believe we should use it here. Can we just add an [upgrade guide entry](https://github.com/GoogleCloudPlatform/magic-modules/blob/main/mmv1/third_party/terraform/website/docs/guides/version_6_upgrade.html.markdown) + maybe mention in documentation description?

Reply:
Understood. I have removed the deprecation_message and added info to docs about the changes coming in 6.0.0.